lib/node_modules/@stdlib/math/base/tools/hermitepoly/README.md

```diff
@@ -0,0 +1,148 @@
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# Physicist's Hermite Polynomial
+
+> Evaluate a physicist's [Hermite polynomial][hermite-polynomial].
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+The physicist's [Hermite polynomials][hermite-polynomial] are given by
+
+<!-- <equation class="equation" label="eq:normalized_hermite_polynomials" align="center" raw="He_{n}(x)=(-1)^{n} e^{\frac{x^2}{2}} \frac{\mathrm d^{n}}{\mathrm d x^{n}} e^{-\frac{x^2}{2}}" alt="Equation for normalized Hermite polynomials."> -->
+
+<div class="equation" align="center" data-raw-text="He_{n}(x)=(-1)^{n} e^{\frac{x^2}{2}} \frac{\mathrm{d}^{n}}{\mathrm{d}x^n} e^{-\frac{x^2}{2}}" data-equation="eq:normalized_hermite_polynomials">
+    <img src="https://cdn.rawgit.com/stdlib-js/stdlib/e1ec71cd46ff5d80734cdba90e8361793260059c/lib/node_modules/@stdlib/math/base/tools/normhermitepoly/docs/img/equation_normalized_hermite_polynomials.svg" alt="Equation for normalized Hermite polynomials.">
+    <br>
+</div>
+
+<!-- </equation> -->
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var hermitepoly = require( '@stdlib/math/base/tools/hermitepoly' );
+```
+
+#### hermitepoly( n, x )
+
+Evaluates a physicist's [Hermite polynomial][hermite-polynomial] of degree `n`.
+
+```javascript
+var v = hermitepoly( 1, 1.0 );
+// returns 2.0
+
+v = hermitepoly( 1, 0.5 );
+// returns 1.0
+
+v = hermitepoly( 0, 0.5 );
+// returns 1.0
+
+v = hermitepoly( 2, 0.5 );
+// returns -1.0
+
+v = hermitepoly( -1, 0.5 );
+// returns NaN
+```
+
+#### hermitepoly.factory( n )
+
+Returns a `function` for evaluating a physicist's [Hermite polynomial][hermite-polynomial] of degree `n`.
+
+```javascript
+var polyval = hermitepoly.factory( 2 );
+
+var v = polyval( 0.5 );
+// returns -0.75
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- Package usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- Package usage examples. -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var randu = require( '@stdlib/random/base/randu');
+var hermitepoly = require( '@stdlib/math/base/tools/hermitepoly' );
+
+var xx;
+var yy;
+var x;
+var y;
+var i;
+var j;
+
+for ( i = 0; i < 100; i++ ) {
+    x = (randu()*100.0) - 50.0;
+    for ( j = 1; j < 3; j++ ) {
+        y = hermitepoly( j, x );
+        xx = x.toFixed(3);
```

Comment:
No need for assigning to variables here. Just inline both calls to `toFixed` in L123.